src/aatoolbox/datasources/usgs/usgs_ndvi.py

```diff
@@ -314,30 +274,35 @@ def process(
         >>> from aatoolbox import create_country_config, \
         ...  CodAB, UsgsNdviSmoothed
         >>>
-        >>> # Retrieve admin 2 boundaries for Nepal
-        >>> country_config = create_country_config(iso3="npl")
+        >>> # Retrieve admin 2 boundaries for Burkina Faso
+        >>> country_config = create_country_config(iso3="bfa")
         >>> codab = CodAB(country_config=country_config)
-        >>> npl_admin2 = codab.load(admin_level=2)
-        >>> npl_admin1 = codab.load(admin_level=2)
+        >>> bfa_admin2 = codab.load(admin_level=2)
+        >>> bfa_admin1 = codab.load(admin_level=1)
         >>>
         >>> # setup NDVI
-        >>> UsgsNdviSmoothed(
+        >>> bfa_ndvi = UsgsNdviSmoothed(
         ...     country_config=country_config,
-        ...     coverage_area="west_africa",
         ...     start_date=[2020, 1],
         ...     end_date=[2020, 3]
         ... )
-        >>> UsgsNdviSmoothed.download()
-        >>> UsgsNdviSmoothed.process()
+        >>> bfa_ndvi.download()
+        >>> bfa_ndvi.process(
+        ...     gdf=bfa_admin2,
+        ...     feature_col="ADM2_FR"
+        ... )
         >>>
-        >>> # process for admin
-        >>> UsgsNdviSmoothed.processed_file_suffix = "adm1"
-        >>> UsgsNdviSmoothed.process()
+        >>> # process for admin1
+        >>> bfa_ndvi.processed_file_suffix = "adm1"
```

Comment:
Is this still needed, as the `feature_col` name is now automatically appended to the filename?

Reply:
NOPE!